microsoft-edge/toc.yml

```diff
@@ -1,327 +1,550 @@
+              href: devtools-guide-chromium/device-mode/testing-other-browsers.md
+              displayName: Device Emulation, Device Emulator
+# Closable tools in alpha order ===============================================
+# 3D View tool ----------------------------------------------------------------
+        - name: 3D View tool
           items:
-            - name: About Panel and Drawer tools
-              href: devtools-guide-chromium/about-panel-drawer-tools.md
-            - name: 3D View
+            - name: Navigate z-index, DOM, and layers using the 3D View tool # added article
               href: devtools-guide-chromium/3d-view/index.md
-            - name: Animations
+              displayName: 3D View tool, Layers tool
+# Animations tool -------------------------------------------------------------
+        - name: Animations tool
+          items:
+            - name: Inspect animations
               href: devtools-guide-chromium/inspect-styles/animations.md
-            - name: Application
-              items:
-                - name: Cache
-                  href: devtools-guide-chromium/storage/cache.md
-                - name: Cookies
-                  href: devtools-guide-chromium/storage/cookies.md
-                - name: Debug background services
-                  href: devtools-guide-chromium/javascript/background-services.md
-                - name: Debug PWAs
-                  href: devtools-guide-chromium/progressive-web-apps/index.md
-                - name: IndexedDB
-                  href: devtools-guide-chromium/storage/indexeddb.md
-                - name: Local storage
-                  href: devtools-guide-chromium/storage/localstorage.md
-                - name: Session storage
-                  href: devtools-guide-chromium/storage/sessionstorage.md
-                - name: WebSQL
-                  href: devtools-guide-chromium/storage/websql.md
-            - name: Changes
+              displayName: Animations tool
+# Application tool (default tab 8) --------------------------------------------
+        - name: Application tool (for storage)
+          items:
+            - name: View, edit, and delete cookies
+              href: devtools-guide-chromium/storage/cookies.md
+              displayName: Application tool
+            - name: View and edit local storage
+              href: devtools-guide-chromium/storage/localstorage.md
+              displayName: Application tool
+            - name: View and change IndexedDB data
+              href: devtools-guide-chromium/storage/indexeddb.md
+              displayName: Application tool
+            - name: View and edit session storage
+              href: devtools-guide-chromium/storage/sessionstorage.md
+              displayName: Application tool
+          # - name: View Application Cache data # deprecated
+          #   href: devtools-guide-chromium/storage/application.md # deprecated
+          #   displayName: Application tool
```

Comment:
I vote for just removing those lines entirely. The tool (and feature it used to debug) is deprecated, and will not come back. So there isn't really a case in the future where we will need these lines.

Reply:
lines kept, for periodic inventory, to avoid re-analysis